integrated_channels/integrated_channel/learner_data.py

```diff
@@ -179,7 +183,7 @@ def _collect_grades_data(self, enterprise_enrollment, course_details):
         except HttpNotFoundError:
             # Grade not found, so we have nothing to report.
             LOGGER.error("No grades data found for %d: %s, %s", enterprise_enrollment.pk, course_id, username)
-            return None, None
+            return None, None, None
```

Comment:
Please document what is being returned by this function in the docstring -- while you're there you can add the argument info as well.